src/_typography.css

```diff
@@ -127,9 +127,10 @@ p {
   content: '';
   display: block;
   height: var(--space-0);
-  margin-left: 0.095rem;
   width: 42px;
   margin-bottom: var(--space-2);
+  margin-left: auto;
```

Comment:
Does this impact letter-rule usage where the text is not centered? e.g. homepage

Reply:
It did.  I added a commit so it'll only target ones with the center class.